app/src/main/java/com/willowtree/vocable/presets/Phrase.kt

```diff
@@ -34,6 +34,7 @@ data class PresetPhrase(
     override val phraseId: String,
     override val sortOrder: Int,
     override val lastSpokenDate: Long?,
+    val deleted: Boolean,
```

Comment:
Thoughts on filtering at the repository level, and then not needing the concept of `deleted` being in any of our domain models?